Backend/editor/models/paragraph.go

```diff
@@ -11,20 +14,25 @@ type Paragraph struct {
 }
 
 type Text struct {
-	text      string
-	link      string
-	bold      bool
-	italic    bool
-	underline bool
+	Text      string
+	Link      string
+	Bold      bool
+	Italic    bool
+	Underline bool
 }
 
 // Get returns the reflect.Value corresponding to a specific field
-func (p Paragraph) Get(field string) (reflect.Value, error) {
+func (p *Paragraph) Get(field string) (reflect.Value, error) {
 	return reflect.ValueOf(p).FieldByName(field), nil
 }
 
 // Set sets a reflect.Value given a specific field
-func (p Paragraph) Set(field string, value reflect.Value) error {
+func (p *Paragraph) Set(field string, value reflect.Value) error {
```

Comment:
Also, should we calling .String more than once? I feel like .String is a rather expensive operation on a reflect.Value as all the processing happens at runtime. It would be better to compute value.String once and reuse it